source/common/protobuf/utility.cc

```diff
@@ -157,6 +157,8 @@ void tryWithApiBoosting(MessageXformFn f, Protobuf::Message& message) {
   } catch (ApiBoostRetryException&) {
     // If we fail at the earlier version, try f at the current version of the
     // message.
+    ENVOY_LOG_MISC(debug,
```

Comment:
again if you'd like a review pass it'd be super helpful if you cleaned up things that aren't part of the PR